internal/controllers/certificaterequest_controller.go

```diff
@@ -28,8 +28,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
-	"k8s.io/apimachinery/pkg/util/clock"
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/utils/clock"
```

Comment:
This module got spun out some time ago. 
Upgrading cert-manager brings a newer version of apimachinery which no longer has the clock package.